jvm-execution/src/main/scala/com/avast/server/toolkit/execution/ConfigurableThreadFactory.scala

```diff
@@ -35,6 +35,7 @@ object ConfigurableThreadFactory {
   /**
     * @param nameFormat Formatted with long number, e.g. my-thread-%02d
     */
+  @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
```

Comment:
why are we calling DefaultArguments wart? if it's a wart, why are we using it?

Reply:
@sideeffffect Can you explain the wart? I am using default arguments on purpose and having to suppress it everywhere in my code seems very strange, I'd rather remove the wart.

Reply:
default arguments are frowned upon in the FP community, I personally avoid them too
see https://blog.ssanj.net/posts/2019-05-01-why-are-default-parameter-values-considered-bad-in-scala.html
but this wart applies also to constructors too, which is a big part of PureConfig functionality
so I'm disabling the wart